__init__.py

```diff
@@ -2,11 +2,10 @@
 # The COPYRIGHT file at the top level of this repository contains
 # the full copyright notices and license terms.
 from trytond.pool import Pool
+from . import sale
 
 def register():
     Pool.register(
+        sale.SaleLine,
+        depends=['sale_discount'],
```

Comment:
In case require custom other modules, not add fix depends; use extras depends.

At the moment, not is necessary add "purchase_discount" depends.